modules/xmpp/JingleSessionPC.js

```diff
@@ -1576,6 +1576,18 @@ export default class JingleSessionPC extends JingleSession {
      */
     replaceTrack(oldTrack, newTrack) {
         const workFunction = finishedCallback => {
+            // Check if the connection was closed and pretend everything is OK.
+            // This can happen if a track removal is scheduled but takes place
+            // after the connection is closed.
+            if (this.peerconnection.signalingState === 'closed'
```

Comment:
But probably 'onTerminated' needs to be executed on the queue, so maybe your version is better for now